src/components/general/FeedDisplay/FeedList.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useEffect, useState, useContext } from 'react';
+import { invoke } from '@tauri-apps/api';
+import { SelectedFeedContext } from '../../contexts/SelectedFeedContext';
+import FeedItem from './FeedItem';
+import FeedSettingsModal from './FeedSettingsModal';
+import styles from './FeedDisplay.module.css';
+
+export default function FeedList() {
+  const [feeds, setFeeds] = useState([]);
+  const [isModalOpen, setIsModalOpen] = useState(false);
+  const [currentFeed, setCurrentFeed] = useState(null); // State for the currently selected feed
+  const { selectedFeed, setSelectedFeed } = useContext(SelectedFeedContext);
+
+  useEffect(() => {
+    invoke('load_feeds')
```

Comment:
This appears to trigger the database load_feeds() function. Does this take care of ticket #23?

Reply:
No, I think this was from awhile ago (only showing here due to other changes). Right now this is the front end requesting the feeds but it still takes as long as the shorted feed poll timer. I think this eventually goes away and then we can move to, instead of a front end request feed, a backend push feeds when loaded and on a timer.